vocode/streaming/models/transcript.py

```diff
@@ -100,29 +100,11 @@ def add_bot_message(self, text: str, conversation_id: str):
             conversation_id=conversation_id,
         )
 
-    def add_vector_db_results(self, text: str, conversation_id: str):
-        self.add_message(
-            text=text,
-            sender=Sender.VECTOR_DB,
-            conversation_id=conversation_id,
-        )
-        last_user_message_idx = self.get_last_user_message()[0]
-        self.event_logs.insert(last_user_message_idx, self.event_logs.pop())
-
     def get_last_user_message(self):
```

Comment:
is this used anywhere? can we delete this too?

Reply:
it is used here: https://github.com/vocodedev/vocode-python/blob/fc5ddf4b5fd1b0beb1fd18a2199eb298c475b449/vocode/streaming/agent/chat_gpt_agent.py#L145